frameworks/helloworld/tests/test_multiple_ports.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import logging
+from tests import config
+
+import sdk_install
+import sdk_plan
+import sdk_tasks
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='module', autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.sanity
+def test_install():
+    sdk_install.install(config.PACKAGE_NAME,
+                        config.SERVICE_NAME,
+                        0,
```

Comment:
Is the number of tasks not equal to `{{HELLO_COUNT}}` on https://github.com/mesosphere/dcos-commons/pull/2593/files#diff-4119b477652218ce29e7fb630f9c13b0R7?

Reply:
The task is not a long running task, by the time our tests see the TASK_RUNNING, it has already moved to TASK_FINISHED and that is why I used 0.